book-service/src/main/java/com/jpop/bookservice/model/Book.java

```diff
@@ -0,0 +1,64 @@
+package com.jpop.bookservice.model;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "books")
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(name = "title")
+    private String title;
+
+    @Column(name = "author")
+    private String author;
+
+    @Column(name = "category")
+    private String  category;
+
+    @Column(name = "description")
+    private String  description;
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public String getAuthor() {
+        return author;
+    }
+
+    public void setAuthor(String author) {
+        this.author = author;
+    }
+
+    public String getCategory() {
+        return category;
+    }
+
+    public void setCategory(String category) {
+        this.category = category;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+}
```

Comment:
You can use lombok annotations to generate constructors, getter and setters wherever possible

Reply:
Sure. In the next upcoming any new Model I will use Lombok. So that I will have both the approaches.